tools/openshift-ci/Dockerfile.centos

```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM quay.io/centos/centos:7
```

Comment:
This is to avoid

> build error: failed to pull image: After retrying 2 times, Pull image still failed due to error: while pulling "docker://centos:7" as "centos:7": Error initializing source docker://centos:7: Error reading manifest 7 in docker.io/library/centos: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit

I'm not sure anybody is using this image but anyway.